test/src/main/java/org/apache/accumulo/harness/conf/AccumuloClusterPropertyConfiguration.java

```diff
@@ -57,33 +57,13 @@ public static AccumuloClusterPropertyConfiguration get() {
     if (propertyFile != null) {
       // Check for properties provided in a file
       File f = new File(propertyFile);
-      if (f.exists() && f.isFile() && f.canRead()) {
-        Properties fileProperties = new Properties();
-        FileReader reader = null;
-        try {
-          reader = new FileReader(f);
-        } catch (FileNotFoundException e) {
-          log.warn("Could not read properties from specified file: {}", propertyFile, e);
-        }
-
-        if (reader != null) {
-          try {
-            fileProperties.load(reader);
-          } catch (IOException e) {
-            log.warn("Could not load properties from specified file: {}", propertyFile, e);
-          } finally {
-            try {
-              reader.close();
-            } catch (IOException e) {
-              log.warn("Could not close reader", e);
-            }
-          }
-
-          clusterTypeValue = fileProperties.getProperty(ACCUMULO_CLUSTER_TYPE_KEY);
-          clientConf = fileProperties.getProperty(ACCUMULO_CLUSTER_CLIENT_CONF_KEY);
-        }
-      } else {
-        log.debug("Property file ({}) is not a readable file", propertyFile);
+      Properties fileProperties = new Properties();
+      try (FileReader reader = new FileReader(f)) {
+        fileProperties.load(reader);
+        clusterTypeValue = fileProperties.getProperty(ACCUMULO_CLUSTER_TYPE_KEY);
+        clientConf = fileProperties.getProperty(ACCUMULO_CLUSTER_CLIENT_CONF_KEY);
+      } catch (IOException e) {
+        throw new RuntimeException("Could not read properties from file: " + propertyFile, e);
```

Comment:
For future reference, there is an UncheckedIOException since Java 8 that works well for situations like this.

Reply:
Oh yeah... I always forget about that guy.